modules/web/src/app/cluster/list/cluster/component.ts

```diff
@@ -137,7 +139,7 @@ export class ClusterListComponent implements OnInit, OnChanges, OnDestroy {
         switchMap(_ =>
           combineLatest([
             this._userService.getCurrentUserGroup(this._selectedProject.id),
-            this._clusterService.restores(this._selectedProject.id),
+            this._enableEtcdBackups ? this._clusterService.restores(this._selectedProject.id) : of([]),
```

Comment:
This might not work if the admin settings are not received before this code is executed. Wdyt?